### PR TITLE
Travis: Update to Go 1.8.

### PR DESCRIPTION
**Pull request by dmitshur**
Remove tip. It will not work, because GopherJS requires the current version of Go.

### CODE REVIEW

**Review by dominikh — APPROVED** [review]

**Comment by dmitshur** [comment]
Oh, actually, while GopherJS requires current version of Go, gopherpen only uses GopherJS compiler when `dev` build tag is used. So it should support tip. I'll bring it back.

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@ sudo: false
 language: go
 go:
   - 1.8.x
+  - master
```
Comment (dominikh) [inline .travis.yml]: Does `master` mean the same thing as `tip` to Travis?

Comment (dmitshur) [reply .travis.yml]: Yes, it's the latest way of specifying it in Travis, that's why I changed it.

Reference: https://docs.travis-ci.com/user/languages/go/#Specifying-a-Go-version-to-use

**Comment by dmitshur** [comment]
PTAL.

**Review by dominikh — APPROVED** [review]

**Pull request merged**